expan/core/experiment.py

```diff
@@ -284,6 +286,12 @@ def _get_weights(self, data, test, variant_name):
         number_of_non_zeros_and_nans = len(x) - number_of_zeros_and_nans
         return number_of_non_zeros_and_nans / np.nansum(x) * x
 
+    def _get_denominators(self, data, test, variant_name):
+        if type(test.kpi) is not DerivedKPI:
```

Comment:
@aaron-mcdaid-zalando I noticed you left _get_weights method, do we need it if we have this one? Also would be nice to have dosctring here.